bot/src/commands.js

```diff
@@ -50,5 +51,7 @@ export async function handleCommands(event, web) {
       timestamp: ts,
       name: 'heavy_check_mark',
     })
+    isCommand = true
```

Comment:
This could also return early if it finds a command. But if I think if more commands match a message, it's more consistent to execute all of them, than only the first one.